### PR TITLE
Fix #59: Added recordVideoDir option

### LINKED ISSUE #59: Allow for recording of test
Hello,

It would be amazing if there was a configuration option for recordings, and where to put them. Bonus points to be able to name recordings after the test name.

### PR DESCRIPTION
**Pull request by melloware**
Fix #59: Added recordVideoDir option

### CODE REVIEW

File: testing/src/main/java/io/quarkiverse/playwright/QuarkusPlaywrightManager.java [hunk 1]
```diff
@@ -69,6 +77,10 @@ private static BrowserType browser(Playwright playwright, WithPlaywright.Browser
 
     @Override
     public void stop() {
+        if (this.playwrightContext != null) {
+            this.playwrightContext.close();
+            this.playwrightContext = null;
+        }
```
Comment (melloware) [inline testing/src/main/java/io/quarkiverse/playwright/QuarkusPlaywrightManager.java:83]: @ia3andy not sure if this was necessary but it felt safe?  I couldn't tell from Microsoft docs whether playwrite.close() closes the context also?

File: integration-tests/pom.xml [hunk 2]
```diff
@@ -9,7 +9,7 @@
     <artifactId>quarkus-playwright-integration-tests</artifactId>
     <name>Quarkus Playwright - Integration Tests</name>
     <properties>
-        <quarkus.version>3.10.1</quarkus.version>
+        <quarkus.version>3.12.0</quarkus.version>
```
Comment (ia3andy) [inline integration-tests/pom.xml:12]: @melloware any reason to upgrade Quakrus? We can stay on web bundler 1.5.x

**Comment by melloware** [comment]
No reason to upgrade. Should I back up to 3.8.5 LTS?

File: integration-tests/pom.xml (hunk 2 above)
Comment (melloware) [reply integration-tests/pom.xml]: I reverted back to 3.10.1 here in the integration test

**Review by ia3andy — APPROVED** [review]
Thanks :)

**Pull request merged**